### PR TITLE
Fix docker images

### LINKED ISSUE #953: Docker image is broken since 2024.5.3.2
**Describe the bug**
The docker run command `docker run -d -v /home/me/myEngine:/lichess-bot/config --name myBot lichessbotdevs/lichess-bot` that worked with 2024.5.3.1 results in the container to fail during startup with the following log content:
```
2024-05-06 18:28:23 /bin/sh: docker/copy_files.sh: Permission denied
```

**To Reproduce**
My configuration folder contains `config.yml` and `homemade.py`.

**Expected behavior**
The container should run as with previous releases.

**Logs**
See above.

**Desktop:**
 - OS: Windows 11 with Docker Desktop
 - Terminal: PowerShell

**Additional context**
As I wrote [here](https://github.com/lichess-bot-devs/lichess-bot/pull/936#issuecomment-2043038504). Having a pre-launch script that copies files in the container file system is not optimal. In my opinion, it is a step in the wrong direction. My advice is to revert the #951 PR and to develop an indirect access to `homemade.py` like I did for `extra_game_handlers.py`.

### PR DESCRIPTION
**Pull request by AttackingOrDefending**
## Type of pull request:
- [x] Bug fix
- [ ] Feature
- [ ] Other

## Description:

Fix copying files in docker.

## Related Issues:

closes #953

## Checklist:

- [x] I have read and followed the [contribution guidelines](/docs/CONTRIBUTING.md).
- [x] I have added necessary documentation (if applicable).
- [x] The changes pass all existing tests.

### CODE REVIEW

File: docker/Dockerfile
Comment (fathzer) [inline docker/Dockerfile:1]: Could you please remove the line that mentioned me as the maintainer of this image?
This line is `MAINTAINER Jean-Marc Astesana <admin@fathzer.com>`

**Comment by MarkZH** [comment]
Is this ready to merge?

File: docker/Dockerfile
Comment (AttackingOrDefending) [reply docker/Dockerfile:1]: Done.

**Comment by AttackingOrDefending** [comment]
Yes.

**Review by MarkZH — APPROVED** [review]

**Pull request merged**